modules/openapi-generator/src/main/resources/csharp/libraries/generichost/modelGeneric.mustache

```diff
@@ -68,6 +78,15 @@
         }
 
         {{/composedSchemas.oneOf}}
+        {{#useObjectInitializerApproach}}
+        {{#hasRequired}}
```

Comment:
Does this have to be behind the new useObjectInitialzerApproach switch? I think you can use net80OrLater instead.

Why is it also behind hasRequired?

Reply:
I put it behind the `hasRequired` switch so it is only generated when the normal constructor does have some mandatory arguments to prevent ambiguity.
And I put it behind the `useObjectInitializerApproach` switch to not generate this constructor without also adding the required keyword to properties. This is necessary to prevent the instantiation of the model without specifying the required fields.

Yes, the `useObjectInitializerApproach` could be replaced by the `net80OrLater` or even the `net70OrLater` switch, but I didn't want to introduce any changes to currently generated code. Should I change it?

Reply:
I haven't used the feature myself, but I don't see any reason to not use net70OrLater since it appears to just be a syntax change.

Reply:
Yes its basically a syntax change and the old syntax can still be used.
The only thing I see that is different using the new approach is, that `OnCreated` is not called. I didn't include it in the parameter less constructor, because it would get called before the required properties are set in contrast to the current constructor where it gets called after all properties are set.